src/pages/Experience/Experience.js

```diff
@@ -1,9 +1,146 @@
 import React from 'react';
+import OpenSourceCard from './open-source-card';
 
 const Experience = () => {
+    const openSourcePrograms = [
+        {
+          title: "GirlScript Summer Of Code 2020",
```

Comment:
[Sample Reference for using assets like images](https://github.com/smaranjitghose/doc2pen/blob/master/src/pages/Home/sections/OpenSource/OpenSource.jsx)